data/shortcodes.csv

```diff
@@ -23,6 +23,7 @@ Lausanne range - 0100 to 07FF,,,
 0116,medframes,"The Medieval Frame of Monumental Portal Sculpture (Pedestals, Canopies, Niches) : Cultural Transfers and Agency (12th-13th Centuries)",
 0118,rome-siecles-obscurs,Rome aux siècles "obscurs". Les lumières de la communication visuelle, Ve-XIe siècles,
 0119,prethero,Premiers théâtres romands,
+0121,societe-savoie,Société de cour, élites et finances : de l’Hôtel des princes de Savoie à l’État moderne,
```

Comment:
```suggestion
0121,societe-savoie,Société de cour élites et finances : de l’Hôtel des princes de Savoie à l’État moderne,
```

Reply:
It is not allowed to have commas inside this comma separated format (otherwise it will be interpreted as separator). I have no idea where this CSV is used - so I don't know all the implications. Would it be possible to just omit the comma (as in my suggested change)?

Reply:
Yes sure. Thanks for checking.